bin/dfx-ckbtc-deploy

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+# Original source: https://github.com/dfinity/ckBTC-Minter-Frontend/blob/master/local_deploy.sh
+# Via:             https://github.com/dfinity/nns-dapp/blob/main/scripts/ckbtc/deploy-ckbtc
+set -euo pipefail
+SOURCE_DIR="$(dirname "$(realpath "${BASH_SOURCE[0]}")")"
+PATH="$SOURCE_DIR:$PATH"
+. "$SOURCE_DIR/versions.bash"
+
+print_help() {
+  cat <<-EOF
+
+	Installs ckbtc canisters.
+
+	EOF
+}
+
+# Source the clap.bash file ---------------------------------------------------
+source "$SOURCE_DIR/clap.bash"
+# Define options
+clap.define short=p long=prefix desc="Prefix for the local canister names" variable=LOCAL_PREFIX default="ckbtc_"
+# Source the output file ----------------------------------------------------------
+source "$(clap.build)"
+
+"${SOURCE_DIR}/dfx-ckbtc-import" --check --prefix "$LOCAL_PREFIX" || {
+  echo "ERROR: ckbtc canister data and files are not all present and correct."
+  echo "       Please run:"
+  echo "       dfx-ckbtc-import --prefix '$LOCAL_PREFIX'"
+  echo
+  exit 1
+} >&2
+
+dfx canister create "${LOCAL_PREFIX}ledger" --network "$DFX_NETWORK"
+dfx canister create "${LOCAL_PREFIX}minter" --network "$DFX_NETWORK"
+
+MINTERID="$(dfx canister id "${LOCAL_PREFIX}minter" --network "$DFX_NETWORK")"
+echo "$MINTERID"
+LEDGERID="$(dfx canister id "${LOCAL_PREFIX}ledger" --network "$DFX_NETWORK")"
+echo "$LEDGERID"
+
+# echo "Step 2: deploying minter canister..."
+dfx deploy "${LOCAL}minter" --network "$DFX_NETWORK" --argument "(variant {
+  Init = record {
+       btc_network = variant { Regtest };
+       ledger_id = principal \"$LEDGERID\";
+       ecdsa_key_name = \"dfx_test_key\";
+       retrieve_btc_min_amount = 0;
+       max_time_in_queue_nanos = 420_000_000_000;
+       min_confirmations = opt 12;
+       mode = variant { GeneralAvailability };
+       kyt_fee = opt 13_333;
+       kyt_principal = null;
+   }
+})" --mode=reinstall
+
+echo "Step 3: deploying ledger canister..."
+PRINCIPAL="$(dfx identity get-principal)"
+dfx deploy "${LOCAL}ledger" --network "$DFX_NETWORK" --argument "(variant {
+  Init = record {
+     token_symbol = \"ckBTC\";
```

Comment:
Should any of this be variable?  Do you need both test ckbtc and main ckbtc and if so, should they have different symbols and token names?

Reply:
@peterpeterparker what do you think?

Is one enough for testnet?

Reply:
Ideally our test environment would deploy ckTESTBTC and - or connect to - a Bitcoin testnet.

ckBTC without the Bitcoin testnet is just another ICRC token as any Sns token.

Reply:
Sounds good.  My impression is that deploying ckTESTBTC should be possible but would mean more code, so it would be better to do that part in a separate PR.  If we aim to deploy ckTESTBTC, is the data in this PR compatible?  E.g. is the token name fine?  We can always adjust later but if it's right from the start we don't risk breaking e2e tests that use the first token name.

If the code here is consistent with the goal, can we merge please?

Reply:
The effort to have `ckTESTBTC` is big. Even the FI team don't have it. Maybe we should wait until they do?

We have a lot of things in our plates at the moment.